x/stakeibc/keeper/lsm.go

```diff
@@ -151,13 +151,20 @@ func (k Keeper) GetValidatorFromLSMTokenDenom(denom string, validators []*types.
 	}
 	validatorAddress := split[0]
 
-	// Confirm validator is in Stride's validator set and does not have an active slash query in flight
+	// Confirm the validator:
+	//  1. is in Stride's validator set
```

Comment:
We are going to expand the stride validator set to include basically all known (6 month+) validators so this will still work on the 0 weighted ones correct?

Reply:
yup, 0 weight vals are included in the set

Reply:
i'll add a clarifying comment